pages/clustering/high-availability.mdx

```diff
@@ -128,7 +129,7 @@ export MEMGRAPH_MANAGEMENT_PORT=10011
 export MEMGRAPH_BOLT_PORT=7692
 ```
 
-When using either of these flags, configurations flags for `--bolt-port`, `--management-port` and `--experimental-enabled` will be ignored.
+When using either of these envronment variables, flags `--bolt-port`, `--management-port` and `--experimental-enabled` will be ignored.
```

Comment:
@kgolubic can you fix this typo?

Reply:
Fixed in https://github.com/memgraph/documentation/commit/0002026276044e905db75dbe5363dbc073f6b339